src/hb-ot-map.hh

```diff
@@ -140,12 +140,12 @@ struct hb_ot_map_t
   void get_stage_lookups (unsigned int table_index, unsigned int stage,
 			  const struct lookup_map_t **plookups, unsigned int *lookup_count) const
   {
-    if (unlikely (stage == UINT_MAX)) {
+    if (unlikely ((stage == UINT_MAX) || (stage > stages[table_index].length)))
```

Comment:
Just remove the first check, as is covered by the second. Thanks. LGTM otherwise. Please merge.